__tests__/workers/cdc/primary.ts

```diff
@@ -5712,3 +5726,289 @@ describe('opportunity match', () => {
     expect(triggerTypedEvent).toHaveBeenCalledTimes(0);
   });
 });
+
+describe('opportunity', () => {
+  it('should trigger on new opportunity', async () => {
+    await expectSuccessfulBackground(
+      worker,
+      mockChangeMessage<OpportunityJob>({
+        after: {
+          id: '2eca06f7-dcaf-4681-8009-c1240c4dd278',
+          createdAt: new Date().getTime(),
+          updatedAt: new Date().getTime(),
+          type: OpportunityType.Job,
+          title: 'Senior Backend Engineer',
+          tldr: 'We are looking for a Senior Backend Engineer...',
+          content: [],
+          meta: {},
+          state: OpportunityState.LIVE,
+          organizationId: '1-cusc',
+        },
+        op: 'c',
+        table: 'opportunity',
+      }),
+    );
+
+    expect(notifyJobOpportunity).toHaveBeenCalledTimes(1);
```

Comment:
I think it's better to test the final "triggeredTypedEvent" just because we might miss some other logic otherwise.